product_docs/docs/epas/15/database_administration/14_edb_clone_schema/cloning_with_a_non_super_user.mdx

```diff
@@ -1,37 +1,37 @@
 ---
-title: Cloning schema as a non-super user
+title: Cloning schema as a non-superuser
 ---
 
-You can now clone the schema as a non-super user. These two functions are created while creating the extension:
+## Cloning schema as a non-superuser
```

Comment:
You can clone the schema as a non-superuser. These two functions are created while creating the extension: